src/gz21_ocean_momentum/trainScript.py

```diff
@@ -161,13 +168,35 @@ def check_str_is_None(string_in: str):
 )
 params = parser.parse_args()
 
+def try_get_forcing_data_filepath(params):
+    """
+    Try to get the forcing data filepath to use from the provided command line
+    options.
+
+    Returns a filepath which should be a zarr dataset. Correctness is not
+    asserted, so the filepath may not exist or may not be a valid zarr dataset.
+
+    TODO bit wonky the way we do mutual exclusion.
```

Comment:
Maybe better put todo into issue and remove here

Reply:
Clarified with comments. I think it's fairly clear now. (I'm disappointed that I can't find a good way to do mutually exclusive arguments directly in argparse!)